src/style.css

```diff
@@ -0,0 +1,31 @@
+
+body {
+  background-color: #000;
+  margin: 20px;
+}
+
+.block-color3 {
+  padding: 10px;
+  width: 350px;
+  margin-bottom: 40px;
+  background-color: #308dc0;
+  border: 1px #e75551 solid;
+  box-sizing: border-box;
+}
+
+.block-color2 {
+  padding: 10px;
+  background-color: #fda70b;
+  border: 1px #e75551 solid;
+  box-sizing: border-box;
+  height: 152px;
+}
+
+.block-color1 {
+  padding: 10px;
+  background-color: #ddd;
+  border: 1px #e75551 solid;
+  box-sizing: border-box;
+  position: relative;
+  bottom: 16px;
+}
```

Comment:
border and padding props are the same in all three classes, move them to the separate class